src/frontend/src/flows/recovery/confirmSeedPhrase.ts

```diff
@@ -38,14 +38,27 @@ const confirmSeedPhraseTemplate = ({
 }) => {
   const copy = i18n.i18n(copyJson);
 
+  let isFistCheckedWord = true;
+
   // All words, where a `Ref` was added if the word needs checking
   const words: Word[] = words_.map((word) => {
+    let isFirstInput = false;
+
     if (word.check) {
       const elem: Ref<HTMLInputElement> = createRef();
       // NOTE: typescript can't follow if word is deconstructed with {...word}
-      return { word: word.word, check: word.check, elem };
+      if (isFistCheckedWord) {
```

Comment:
can you find another way to solve this? mixing imperative and declarative code is a bit an anti-pattern and therefore prone to error

Reply:
I agree with @peterpeterparker; I think using [`words.find()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) would be cleaner

Reply:
@nmattia I think its cleaner now. Let me know what you think.